src/components/pages/about.tsx

```diff
@@ -0,0 +1,3 @@
+export default function About() {
+    return <h1>About</h1>
```

Comment:
Something minor, let's follow a modern file naming convention:
- `.tsx` files should have words capitalize with no spaces (ex: `Navbar.tsx`, `NavbarElements.tsx`, `About.tsx`)
- `.css` files can be camel-case (ex: `globalStyles.css`, `styles.css`)